docs/releases/6.0.txt

```diff
@@ -324,3 +324,5 @@ to remove usage of these features.
 
 * The ``OS_OPEN_FLAGS`` attribute of
   :class:`~django.core.files.storage.FileSystemStorage` is removed.
+
+* The ``AdminSite.password_change_form`` attribute is now customizable.
```

Comment:
This is currently in the features removed section
It should be moved further up, e.g.
```diff
--- a/docs/releases/6.0.txt
+++ b/docs/releases/6.0.txt
@@ -48,7 +48,8 @@ Minor features
 :mod:`django.contrib.admindocs`
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-* ...
+* The new :attr:`.AdminSite.password_change_form` attribute allows customizing
+  the form used in the admin site password change view.
 
```